api/src/main/java/jakarta/data/page/PageRequest.java

```diff
@@ -415,6 +415,13 @@ static <T> PageRequest<T> ofSize(int maxPageSize) {
      */
     PageRequest<T> sortBy(Sort<T> sort1, Sort<T> sort2, Sort<T> sort3, Sort<T> sort4, Sort<T> sort5);
 
+    /**
+     * Convert this object to an equivalent {@link Limit}.
+     */
+    default Limit asLimit() {
+        return Limit.range( 1 + (page()-1) * size(), size() );
+    }
+
```

Comment:
This doesn't seem related. My guess is that it was accidentally included in this pull.
```suggestion
```

Reply:
Yes, it was an accident, sorry.